### PR TITLE
node/p2p: increase TESTNET_BOOTSTRAP_DHI to 350

### PR DESCRIPTION
**Pull request by tbjump**

### CODE REVIEW

**Comment by tbjump** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3311** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3311" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3311?utm_source=stack-comment).

**Review by evan-gray — APPROVED** [review]
nice comment

File: node/pkg/p2p/p2p.go [hunk 1]
```diff
@@ -46,6 +46,9 @@ const DefaultPort = 8999
 const P2P_VALIDATE_QUEUE_SIZE = 1024
 const P2P_SUBSCRIPTION_BUFFER_SIZE = 1024
 
+// TESTNET_BOOTSTRAP_DHI configures how many nodes may connect to the testnet bootstrap node. This number should not exceed HighWaterMark.
```
Comment (panoel) [inline node/pkg/p2p/p2p.go:49]: Should we `assert(TESTNET_BOOTSTRAP_DHI < HighWaterMark)` if we want to enforce that?

Comment (tbjump) [reply node/pkg/p2p/p2p.go:49]: we could, but we don't usually do that elsewhere in the guardian. I'd love it if there was some way to do that at compile time, but I don't think there is.

**Review by panoel — APPROVED** [review]

**Pull request merged**